components/dataViews/TransactionInfo.tsx

```diff
@@ -16,19 +16,33 @@ const TransactionInfo = (props: Props) => {
   return (
     <StackableContainer lessPadding lessMargin>
       <ul className="meta-data">
-        {props.tx.msgs && (
-          <li>
-            <label>Amount:</label>
-            <div>{printableCoins(props.tx.msgs[0].value.amount, state.chain)}</div>
-          </li>
+        {props.tx.msgs && props.tx.msgs[0].typeUrl === "/cosmos.bank.v1beta1.MsgSend" && (
```

Comment:
Could you loop through all messages to make the code ready for multiple messages? Something like `(props.tx.msgs || []).foreach(…)`